src/main/java/org/onedatashare/server/controller/UserController.java

```diff
@@ -74,6 +74,8 @@ public Object performAction(@RequestHeader HttpHeaders headers, @RequestBody Use
         return userService.isAdmin(cookie);
       case "resendVerificationCode":
         return userService.resendVerificationCode(userAction.getEmail());
+      case "updateViewPreference":
+        return userService.updateViewPreference(userAction.getEmail(), userAction.isCompactViewPreferred());

```

Comment:
Using the word 'enabled' instead of 'preferred' would be better as per standards. 
Please update all occurrences.